purchase_price_recalculation/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,10 @@
+Allows to recompute purchase lines clicking on `Purchase recalculation` entry in purchase action button
+
+Some use cases where this module can be useful:
+
+- product tax have been updated
```

Comment:
Not tested, but the recomputation is not done, when tax has changed ?

Reply:
tax changed on product, not on purchase line